.github/scripts/svgo.config.js

```diff
@@ -0,0 +1,53 @@
+module.exports = {
+  plugins: [
+    {
+      name: 'preset-default'
+    },
+    {
+      name: 'removeDimensions',
+      params: {
+        overrides: {
+          removeDimensions: true,
+        }
+      }
+    },
+    {
+      name: 'convertStyleToAttrs',
+      params: {
+        overrides: {
+          convertStyleToAttrs: true,
+        }
+      }
+    },
+    {
+      name: 'removeRasterImages',
+      params: {
+        overrides: {
+          removeRasterImages: true,
+        }
+      }
+    },
+    {
+      name: 'removeScriptElement',
+      params: {
+        overrides: {
+          removeScriptElement: true,
+        }
+      }
+    },
+    {
+      name: "removeAttrs",
+      params: {
+        attrs: [
+          'path:fill:none',
+          'xml.space',
+          'enable-background',
+          '^data.+',
+          'stroke.*',
+          'fill-rule',
+          'clip-rule'
```

Comment:
`removeDimensions` does it, removing `height` and `width`, and add `viewBox` attribute if is missing.

About second questions. My intention was to read some feedback about them. Then we can convert the settings to a more basic state.

Strokes shouldn't be allowed, but we should be able to see the error first, so I'll be doing some adjustments.

I'll preserve just `path:fill:none` from this plugin, since it cause no problem at all.